### ISSUE TITLE
Issue 2596  WARNING: '/sbin/iptables-restore --wait=2 -n' failed: /sbin/iptables-restore: unrecognized option '--wait=2'  iptables-restore: line 93 failed

### ISSUE THREAD

**Issue by hffej01** [firewalld/firewalld#756]
Feb 09 20:56:31 ****740 firewalld[2596]: WARNING: '/sbin/iptables-restore --wait=2 -n' failed: /sbin/iptables-restore: unrecognized option '--wait=2'

Looking at the file that listed above @ /sbin/iptables-restore   shows this at line 93

And searching for this on sublime, produces nothing    '--wait=2'

----------------------------------------------------------------
Bad line %u: need :
-t--table-AACCEPTDROPQUEUERETURNCombine action from next rule      <rule        <conditions>       </conditions>       <actions>%s: line %u failed
</iptables-rules>verbosecombinehelp1.6.1Usage: %s [-c] [-v] [-h]
          [--combine ]
           [ --verbose ]
           [ --help ]    <<<< Line 93 of the iptables-restore file.

Unknown arguments found on commandline<iptables-rules version="1.0">%s: line %u table name invalid
%s: line %u chain name invalid
%s: line %u failed - no chain found
---------------------------------------------------------------------------------

A new rule was added to the FW on Sunday 02/07/2021 to allow another server to access this server.
firewall-cmd --add-rich-rule='rule family="ipv4" source address="10.X.X.X" accept'   

As of Monday 02/08/2021 night the server was rebooted without any issues.   On Wednesday 02/10/2021  the firewalld stopped working  and blocked everyone.   We finally turned off firewalld and was able to regain access.  

Any suggestions .....

**Comment by erig0** [comment]
This error is coming from `iptables-restore`. It's reporting that `--wait` is not a valid option. When firewalld start it probes `iptables-restore` for which wait options are supported. When firewalld was started it must have detected that `--wait` is available. If `iptables-restore` was upgraded/replaced/downgraded while firewalld was running and firewalld was not restarted, then that could be the cause.

Have you tried restarting firewalld, i.e. `systemctl restart firewalld` ? Does the issue still occur?

**Comment by hffej01** [comment]
Yes, we did a restart on it.   That's when everyone was kicked off the server.    And it has been rebooted a few times since Monday night till now Wednesday morning.

**Comment by hffej01** [comment]
And to the best of my knowledge .....that script has not been touched and or modified.

**Comment by erig0** [comment]
Can you try using the `--wait=2` option directly with `iptables-restore` ?
e.g.
```
# echo "" | iptables-restore --wait=2
```

**Comment by hffej01** [comment]
when added...  echo "" | iptables-restore --wait=2

iptables-restore: unrecognized option '--wait=2'

BUT..... 
We think we found the issue.   When the NEW family rule was added...Family was misspelled  "familt"  instead of a " y" .  

The question still remains ....why did it take 3 days to fail.  And why was an error not produced when first restarted?  


I will leave this open until Monday Feb 15, 2021, to make sure that no other issues come up.
